src/components/Clothing.js

```diff
@@ -1,14 +1,22 @@
 import styled from 'styled-components/macro'
 
-function Clothing({ imageUrl }) {
+function Clothing({ titel, color, pattern, material, fitting, imageUrl }) {
```

Comment:
Maybe you can change 'titel' to 'title' as all the other props are named in English.

Reply:
Thanks for the hint :)